easybuild/tools/options.py

```diff
@@ -1047,8 +1047,50 @@ def _postprocess_checks(self):
 
         self.log.info("Checks on configuration options passed")
 
+    def get_cfg_opt_abs_path(self, opt_name, path):
+        """Get path value of configuration option as absolute path."""
+        if os.path.isabs(path):
+            abs_path = path
+        else:
+            abs_path = os.path.abspath(path)
+            self.log.info("Relative path value for '%s' configuration option resolved to absolute path: %s",
+                          path, abs_path)
+        return abs_path
+
+    def _ensure_abs_path(self, opt_name):
+        """Ensure that path value for specified configuration option is an absolute path."""
+
+        opt_val = getattr(self.options, opt_name)
+        if opt_val:
+            if isinstance(opt_val, string_type):
+                setattr(self.options, opt_name, self.get_cfg_opt_abs_path(opt_name, opt_val))
+            elif isinstance(opt_val, list):
+                abs_paths = [self.get_cfg_opt_abs_path(opt_name, p) for p in opt_val]
```

Comment:
But then the whole of `_ensure_abs_path` would basically be collapsed into `get_cfg_opt_abs_path`.

The point of `get_cfg_opt_abs_path` is to handle a *single* path, so it can also be used to handle the `repositorypath` special case.

I think it even makes sense to have multiple log messages for configuration options that are a list of paths.
The log message was carefully worded for that too, it never states that the absolute path set is *the* (only) value for that specific configuration option...

Reply:
True, I just didn't like the idea of getting multiple messages for a single option... Not a big deal though.